packages/rancher-logging/generated-changes/overlay/templates/loggings/kube-audit/logging.yaml

```diff
@@ -0,0 +1,100 @@
+{{- if .Values.additionalLoggingSources.kubeAudit.enabled }}
+{{- $auditLogDir := default "/var/log/kube-audit" .Values.additionalLoggingSources.kubeAudit.logDirectory }}
```

Comment:
Will this change based on provider? Will EKS change dirs compared to RKE2, for example?

Reply:
My thinking was to focus on rke/rke2/k3s initially.  I'm not sure whether we even have access to controlplane stuff for hosted providers, I know with AKS we definitely don't.

Reply:
We do not. Regardless, I think the `values.yaml` field will suffice.